lib/rules/no-deprecated.js

```diff
@@ -73,6 +76,10 @@ module.exports = {
       deprecated[`${pragma}.PropTypes`] = ['15.5.0', 'the npm module prop-types'];
       // 15.6.0
       deprecated[`${pragma}.DOM`] = ['15.6.0', 'the npm module react-dom-factories'];
+      // 16.3.0
+      deprecated.componentWillMount = ['16.3.0'];
```

Comment:
can we supply messages in all these three, that ideally suggest replacements and link to the react docs on the deprecation?

Reply:
Could you take a look at the new message pattern?